cmd/flux/get.go

```diff
@@ -150,6 +153,21 @@ func (get getCommand) run(cmd *cobra.Command, args []string) error {
 		listOpts = append(listOpts, client.MatchingFields{"metadata.name": args[0]})
 	}
 
+	if getArgs.labelSelector != "" {
+		label, err := metav1.ParseToLabelSelector(getArgs.labelSelector)
+		if err != nil {
+			return err
```

Comment:
I personally prefer wrapping errors and provide some context before returning them. But that may be personal preference.

Reply:
Yes let's return here something like `invalid label expression, error: %w`